Ally4/Portfolio
# Setting up the working environment for nodejs

# Description

This PR contains the setting environment of nodejs

## Type of change

Please check the options that are relevant.


# How should this be manually tested or reproduced?

- Clone this repo
- Checkout to `server`
- Open the package.json to see the installation

# Checklist:

- [X] Creating a working server
- [X] Installing and initializing npm 
- [X] Installing babels for transpile
- [X] Installing mongoose for the database
- [X] Installing JSON web token for authentication
- [X] Installing joi for validation
- [X] Installing express as framework
- [X] Installing bcrypt for the encryption of the password
- [X] Installing nodemon for an instant loading
- [X] Creating a .babelrc file
- [X] Adding in .gitignore
